README.md

```diff
@@ -33,14 +33,50 @@ Deployment of a third-party solution requires a PR for a FluxCD deployment submi
 
 ## Pre-requisite (Linux/MacOS)
 
-This solution requires Flux CLI locally and Flux Controller on your Kubernetes cluster. Please follow the below steps for installing these pre-requisites :
+This solution requires Flux CLI locally and Flux Controller on your Kubernetes cluster. Flux expects to access a source repository and this repository, the api to sync the source code, must be reachable. Please follow the below steps for installing these pre-requisites.
+
+If you are staring from scratch, i.e. you do not have a kubernetes cluster sitting around waiting for you, you can use a kubernetes distribution such as [k3s](https://k3s.io/) to set up a kubernetes cluster on the system you want to use for development/testing.
+
+```bash
+curl -sfL https://get.k3s.io | sh -
+```
+
+This will setup a single node kubernetes cluster on your system. Flux integrates into your running cluster and needs to find the configuration file for tha cluster you want to use for testing. Flux will look in the default location, i.e. *~/.kube/config*. 
+
+Before setting up Flux make sure your configuration file points to the cluster you want to used for testing. You can use
+
+```bash
+helm ls --all-namespaces
+```
+
+for example to verify that a suitable kubeconfig file can be found and the cluster can be accessed. If no configuration is found you will get an error message indicating that *"http://localhost:8080/version"* cannot be accessed. Do not be confused by the port number. The port number for accessing the kubernetes cluster is part of the configuration file and the reported port in the error message is a default port.
+
+You can use
+
+```bash
+export KUBECONFIG=$PATH_TO_kubeconfig.yaml
+```
+
+to ensure the flux installation finds the cluster you want to use for testing.
+
+If you installed k3s, as root user, the configuration will be in */etc/rancher/k3s/k3s.yaml* as such
+
+```bash
+export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
+```
+
+will do the trick.
+
+Once you have a kubernetes cluster running and the configuration file is properly setup you are ready to install flux.
+
 
 ```bash
 git clone https://github.com/aws-samples/eks-anywhere-addons.git
 cd eks-anywhere-addons
 chmod +x installFlux.sh
 ./installFlux.sh
 ```
+**Note:** In oreder to commit back to th eproject you will need to create a fork in GitHub into your own project. You will want to clone your fork instead of the repor from the AWS project *https://github.com/aws-samples/eks-anywhere-addons.git*
```

Comment:
fix `th eproject`

Reply:
fix `You might want...`

Reply:
change to `need to create a fork in GitHub pointing to your own project.`

Reply:
Change `You will want to clone your fork instead of the repor from the AWS project *https://github.com/aws-samples/eks-anywhere-addons.git*` to `You might want to clone your fork in the above git clone command instead of cloning the AWS repository *https://github.com/aws-samples/eks-anywhere-addons.git*`